src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -29,63 +32,88 @@ public class FindCommandTest {
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
-
-        FindCommand findFirstCommand = new FindCommand(firstPredicate);
-        FindCommand findSecondCommand = new FindCommand(secondPredicate);
-
+        FindCommandParser parser = new FindCommandParser();
+        FindCommand findFirstCommand;
+        FindCommand findSecondCommand;
+        FindCommand findFirstCommandCopy;
+        try {
+            findFirstCommand = parser.parse("n/first");
+            findSecondCommand = parser.parse("n/second");
+            findFirstCommandCopy = parser.parse("n/first");
+        } catch (Exception e) {
+            e.printStackTrace();
+            return;
+        }
```

Comment:
Replace the Exception with a more specific unchecked exception proper. This might be old code that is not supposed to be here